include/zephyr/net/net_ip.h

```diff
@@ -203,6 +204,13 @@ struct sockaddr_ll {
 	uint8_t     sll_addr[8];  /**< Physical-layer address, big endian */
 };
 
+/* Socket address struct for UNIX domain sockets */
+#define UNIX_PATH_MAX 108
```

Comment:
This likely needs be conditionally set based on `CONFIG_NET_NATIVE_OFFLOADED_SOCKETS`

Reply:
I've moved this define in the flagged section for `CONFIG_NET_NATIVE_OFFLOADED_SOCKETS` that defines `NET_SOCKADDR_MAX_SIZE`.

Then this original definition of `struct sockaddr_un` has been restored. In case of `CONFIG_NET_NATIVE_OFFLOADED_SOCKETS`, the `sun_path` will get the new size.